feat: added publish guidelines for pkgjs owned packages (#419)
nodejs/package-maintenance: Governance.md

of Directors approval may also be required. (Most likely in the case of
transfeering a repository into or out of the organization).

##### Publishing Packages

> When publishing packages owned by the pkgjs org, the maintainers of the project

Reviewer: ```suggestion
When publishing packages owned by the [pkgjs](https://github.com/pkgjs) org, the maintainers of the project
```

can choose to use the global scope or the `@pkgjs` scope on the registry. When

> the package is first published, we always turn on required 2FA to publish. In

Reviewer 2: ```suggestion
the package is first published, we always turn on "required 2FA to publish". In
```

Is this the literal text? How about quotes so its clear what to look for in the UI?

Author: The exact text is "Require two-factor authentication to publish".  I like using that, but npm could always change it, so I wouldn't worry too much about it other than stating we turn that feature on.

```suggestion
the package is first published, we always turn on "Require two-factor authentication to publish". In
```

> can choose to use the global scope or the `@pkgjs` scope on the registry.  When
> the package is first published, we always turn on required 2FA to publish.  In

Reviewer 3: ```suggestion
can choose to use the global scope or the `@pkgjs` scope on the registry. When
the package is first published, we always turn on required 2FA to publish. In
```

Author: Ah yes, despite how hard I try....I cannot escape the muscle memory of "two spaces after a sentence" being taught to me my entire education.

the case of using the global scope, you also need to transfer ownership of the

> package so the `@pkgjs` org on npm.

Reviewer 2: ```suggestion
package to the `@pkgjs` org on npm.
```

Is this a typo?
